templates/openstack/what-is-openstack.html

```diff
@@ -2,206 +2,593 @@
-      <p>Fully integrated with Nova, Cinder supports integration with a wide range of storage platforms.</p>
+      <p class="p-card__content">Although designed for traditional VMs, OpenStack is fully capable of provisioning bare metal machines on-demand too, turning your data centre into a bare metal cloud.</p>
+      <br />
+      <p>
+        <a class="p-link--external" href="https://maas.io">Read more about bare metal cloud</a>
+      </p>
     </article>
     <article class="col-6 p-card">
-      <h3 class="p-card__image p-heading--four">Swift</h3>
+      <h3 class="p-card__title p-heading--three">Containers</h3>
       <hr class="u-sv1" />
-      <p>Swift is a storage system for objects and files. Swift works in a different way to the traditional method of referring to files by their disc drive location, instead it is possible for developers to refer to a unique identifier that refers to the file of piece of information, letting OpenStack decide where this information is stored.</p>
-      <p>One of the main benefits of this is the ease of use it creates for scaling. By removing the need for developers to worry about the capacity on a single system behind the software. Another added advantage is that the system takes care of ensuring that the data is backed up, rather than relying upon the developer to do so.</p>
+      <p class="p-card__content">OpenStack can be easily extended with a container layer running on top of it for better workloads granularity as required by cloud-native applications.</p>
+      <br />
+      <p>
+        <a href="/kubernetes">Read more about containers&nbsp;&rsaquo;</a>
+      </p>
     </article>
   </div>
 </section>
 
-<section class="p-strip is-bordered">
+<section class="p-strip--light">
   <div class="row">
-    <div class="col-8">
-      <h2>Who uses OpenStack?</h2>
-      <p>There are thousands of users of OpenStack across a broad range of industries. Particularly favoured for its security, scalability, resilience.</p>
-      <p>In addition to CERN, the UK Civil Service, and numerous universities around the world, OpenStack has become a popular choice for telecommunication companies such as ATT, Verizon, and China Mobile.</p>
-      <p>Use cases are also growing in the media area with SkyTV and DirecTV having already adopted.</p>
-      <p>Financial services have been a keen adopter, with BBVA, Banco Santander, Commonwealth Bank, and China UnionPay just some that have adopted.</p>
-      <p>Further adoption has been seen in healthcare, manufacturing, and retail.</p>
+    <div class="col-9">
+      <h2>Why Canonical’s Charmed OpenStack?</h2>
+      <p>Wondering which OpenStack platform to choose? This is why Canonical’s Charmed OpenStack is the answer:
+        <ul class="p-list">
```

Comment:
a `<ul>` element shouldn't go inside a `<p>` - see here https://bytes.com/topic/html-css/answers/153770-acceptable-put-ul-inside-paragraph. You can close the `<p>` before hand.